config/core/test-configs.yaml

```diff
@@ -1650,6 +1657,20 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: asus-C523NA-A20057-coral
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - cros-ec
+      - igt-gpu-i915
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - ltp-crypto
+      - ltp-ipc
+      - ltp-mm
+      - sleep_mem
```

Comment:
Tests are showing as "passing" https://staging.kernelci.org/test/plan/id/60a5afb2efe8810d25a35858/

But in fact the job failed: https://storage.staging.kernelci.org/kernelci/staging-mainline/staging-mainline-20210520.0/x86_64/x86_64_defconfig+x86-chromebook/gcc-8/lab-collabora/sleep-asus-C523NA-A20057-coral.html